packages/components/psammead-most-read/src/utilities/rankMinWidth.js

```diff
@@ -59,3 +57,22 @@ export const doubleDigitThin = {
   group3WithTwoColumns: GEL_SPACING_SEXT,
   group5WithFiveColumns: GEL_SPACING_SEXT,
 };
+
+export const singleDigitSmall = {
+  group0: GEL_SPACING_DBL,
+  group1: GEL_SPACING_DBL,
+  group2: GEL_SPACING_DBL,
+  group3: GEL_SPACING_TRPL,
+};
+
+export const doubleDigitSmall = {
```

Comment:
I opted to replicate the data structure above for this new group (`smallFontServices`) as I wouldn't have to change much here: `packages/components/psammead-most-read/src/Rank/index.jsx`